docs/shared/asset/script/docs-example.es

```diff
@@ -5,9 +5,7 @@ angular.module('docs')
     return {
         restrict: 'A',
         link(scope, element, attrs) {
-            scope.$watch(() => {
-                return scope.$eval(attrs.bindHtmlCompile);
-            }, (newVal, oldVal) => {
+            scope.$watch(attrs.bindHtmlCompile, (newVal, oldVal) => {
```

Comment:
@NathanRodger does this really do what you want it to?

Reply:
@Decad Yes it resolved the issue with infinite loop